designs/dragonphy_top/synopsys-dc-synthesis/configure.yml

```diff
@@ -70,7 +71,7 @@ postconditions:
 
   # Basic error checking
 
-  - assert 'error' not in File( 'logs/dc.log' )
+  - assert 'Error:' not in File( 'logs/dc.log' )
```

Comment:
Just wondered what issue motivated this change.  I see that the latest version of **mflowgen** is using the error string ``error:`` (lowercase ``e``), so I wanted to make sure we're still catching all the errors we used to catch.
ref: https://github.com/cornell-brg/mflowgen/blob/f646da41b57b7894fc4e8ef852eb0cd713e97cb3/steps/synopsys-dc-synthesis/configure.yml#L93